build/nsm-init/docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1-alpine as build
+ENV PACKAGEPATH=github.com/ligato/networkservicemesh/
+COPY [".","/go/src/${PACKAGEPATH}"]
+WORKDIR /go/src/${PACKAGEPATH}/
+RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-X main.version=$(REV) -extldflags "-static"' -o /go/bin/nsm-init ./cmd/nsm-init/nsm-init.go
```

Comment:
Where does this REV variable come from? Doesn't seem to be available or set in our scripts.